microsoft/testsuites/rust_vmm_mshv/rust_vmm_mshv_test.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import re
+from pathlib import Path
+
+from assertpy.assertpy import assert_that
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    notifier,
+)
+from lisa.messages import SubTestMessage, TestStatus, create_test_result_message
+from lisa.testsuite import TestResult
+from lisa.tools import Cargo, Git
+from lisa.util.process import ExecutableResult
+
+
+@TestSuiteMetadata(
+    area="rust-vmm",
+    category="community",
+    description="""
+    This test suite is for executing the rust-vmm/mshv tests
+    """,
+)
+class RustVmmTestSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            Runs rust-vmm/mshv tests
+        """,
+        priority=3,
+        timeout=1800,
+    )
+    def verify_rust_vmm_mshv_tests(
```

Comment:
Is the test targeted only for linux dom0 ? 
If so, please add before case and raise SKIPPED exception - https://github.com/microsoft/lisa/blob/main/microsoft/testsuites/mshv/mshv_root_tests.py#L44

Reply:
Done